### PR TITLE
enable creating table without prefix

### LINKED ISSUE #306: [JSTC-27] Don't require prefix flag for create command with schema
## Summary

The `tableland create` command *requires* a `--prefix` to be specified with a `schema`, which is not an actual requirement for creating a table. But, if I do not want to use a prefix, it can be a bit confusing how to make this happen since you need to explicitly do this: `--prefix '""'` 

## Details

Try creating a table with the following command:

```
tableland create "id int" --prefix ""
```

It will error out and potentially cause end user confusion:

```
error parsing statement: syntax error at position 15 near '('
CREATE TABLE  (id int)
```

Instead, it'd like to be able to create a table without having to use the `--prefix` flag altogether, thus, assuming an empty string should be used as the prefix. In other words, the CLI should do `CREATE TABLE "" ...` if there is no prefix flagged.

## Alternatives

Ultimately, this could be handled in docs only (note: already adding this), but it'd be nice if it "just worked" without the `--prefix` flag.

<sub>From [SyncLinear.com](https://synclinear.com) | [JSTC-27](https://linear.app/tableland/issue/JSTC-27/dont-require-prefix-flag-for-create-command-with-schema)</sub>

### PR DESCRIPTION
**Pull request by joewagner**
fixes #306

### CODE REVIEW

File: src/commands/create.ts [hunk 1]
```diff
@@ -103,12 +103,6 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
       logger.error("the `create` command can only accept create queries");
       return;
     }
-    if (statements.length < 1) {
-      logger.error(
-        "after normalizing the statement there was no create query, hence nothing to do"
-      );
```
Comment (joewagner) [inline src/commands/create.ts:109]: Without the possibility of a missing prefix this check isn't needed anymore

File: src/commands/create.ts [hunk 2]
```diff
@@ -69,11 +70,10 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     const check = /CREATE TABLE/gim.exec(schema.toString());
     if (check) {
       statement = schema;
-    } else if (prefix === undefined) {
-      logger.error(
-        "Must specify --prefix if you do not provide a full Create statement"
-      );
     } else {
+      if (prefix == null || prefix.trim() === "") {
```
Comment (joewagner) [inline src/commands/create.ts:74]: if `check` is false and prefix doesn't exist, we will create a table without a prefix.

File: src/commands/create.ts [hunk 3]
```diff
@@ -135,7 +129,7 @@ export const handler = async (argv: Arguments<Options>): Promise<void> => {
     const link = getLink(chain, res.meta.txn?.transactionHash as string);
     const out = { ...res, link, ensNameRegistered: false };
 
-    // TODO: I'm not sure how `check` would be false and statements.length < 2
+    // TODO: I'm not sure how `check` would be false and statements.length > 2
```
Comment (joewagner) [inline src/commands/create.ts:132]: This is a typo left over from a previous commit.

Comment (carsonfarmer) [reply src/commands/create.ts:132]: Should we just remove it?

**Review by carsonfarmer — DISMISSED** [review]

File: src/commands/create.ts (hunk 3 above)
Comment (joewagner) [reply src/commands/create.ts:132]: Good call

Comment (joewagner) [reply src/commands/create.ts]: I removed the comment, but need another 👍

**Review by carsonfarmer — APPROVED** [review]
Nice fixes here @joewagner! You are crushing it!

**Pull request merged**